### PR TITLE
Only users with organizer privilege can see place creation

### PR DESCRIPTION
**Pull request by Mayuresh0072**
This fixes the issue #3700 .

### CODE REVIEW

**Comment by Mayuresh0072** [comment]
Hello @kueda , I have made code changes for solving the issue #3700  .Please have a look at it . Thanks.

**Review by kueda — CHANGES_REQUESTED** [review]
There is no `:organizer` role so this condition will always be false and no one will be able to access this link, which doesn't address the first todo in https://github.com/inaturalist/inaturalist/issues/3700. "Organizer" is a `UserPrivilege`, not a `Role`. You can control what users have what privileges using the admin interface in your local environment.

You also need to address the second todo in https://github.com/inaturalist/inaturalist/issues/3700

File: app/controllers/places_controller.rb [hunk 1]
```diff
@@ -51,17 +51,9 @@ class PlacesController < ApplicationController
...
-      # organizer privilege
-      if @place.blank?
-        true
-      else
-        current_user.id != @place.user_id
-      end
-    }
+  requires_privilege :organizer, only: [:new, :create, :edit, :update, :destroy] do
+    redirect_back_or_default(root_url)
+  end
```
Comment (kueda) [inline app/controllers/places_controller.rb:56]: Why are you removing the ability to edit a place you created regardless of privilege? Also, requires_privilege does not take a block. You can find the full definition at https://github.com/inaturalist/inaturalist/blob/main/lib/privileges.rb#L63

**Review by kueda — CHANGES_REQUESTED** [review]

File: app/controllers/places_controller.rb (hunk 1 above)
Comment (Mayuresh0072) [reply app/controllers/places_controller.rb:56]: @kueda Actually I thought the if block checks whether other users who have not created the place have the edit access. So, now i will keep the 'if block' as it is. But what exactly I have to do for the 2nd Todo task.Can You please explain?

Comment (Mayuresh0072) [reply app/controllers/places_controller.rb:56]: @kueda So, I think I should only add ':new' and ':create' in the 'only:' array of the organizer privilege.

**Review by kueda — APPROVED** [review]

**Comment by pleary** [comment]
I merged the PR as it stands, which addresses the first TODO in #3700. As @kueda noted, the second TODO remains unresolved so far

**Pull request merged**